File: CampingNeretva/CampingNeretva.Model/ReservationRentableItemModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model
{ 
    public class ReservationRentableItemModel
    {
        //public int RentableItemId { get; set; }
        public int Quantity { get; set; }
        public RentableItemModel Item { get; set; }
    }
}

File: CampingNeretva/UI/campingneretva_desktop/lib/models/reservation_rentable_item_model.dart
import 'rentable_item_model.dart';

class ReservationRentableItemModel {
  final int quantity;
  final RentableItem item;

  ReservationRentableItemModel({required this.quantity, required this.item});

  factory ReservationRentableItemModel.fromJson(Map<String, dynamic> json) {
    return ReservationRentableItemModel(
      quantity: json['quantity'],
      item: RentableItem.fromJson(json['item']),
    );
  }
}
